apps/game_backend/lib/game_backend/curse_of_mirra/quests.ex

```diff
@@ -72,51 +90,51 @@ defmodule GameBackend.CurseOfMirra.Quests do
   end
 
   @doc """
-  Receives a DailyQuest id and returns a %DailyQuest{}
+  Receives a UserQuest id and returns a %UserQuest{}
 
   ## Examples
 
       iex>get_daily_quest(daily_quest_id)
-      %DailyQuest{}
+      %UserQuest{}
 
   """
   def get_daily_quest(daily_quest_id) do
-    q = from(dq in DailyQuest, preload: [:quest], where: dq.id == ^daily_quest_id)
+    q = from(dq in UserQuest, preload: [:quest], where: dq.id == ^daily_quest_id)
 
     Repo.one(q)
   end
 
   @doc """
-  Receives a User id and returns all %DailyQuest{} that belongs to that user
+  Receives a User id and returns all %UserQuest{} that belongs to that user
 
   ## Examples
 
       iex>get_users_daily_quests(user_id)
-      [%DailyQuest{}]
+      [%UserQuest{}]
 
   """
   def get_users_daily_quests(user_id) do
-    q = from(dq in DailyQuest, preload: [:quest], where: dq.user_id == ^user_id)
+    q = from(dq in UserQuest, preload: [:quest], where: dq.user_id == ^user_id)
```

Comment:
This is getting all UserQuests (bounties too). The name is wrong

Reply:
This is not being used, i'll just delete the `get_users_daily_quest` function, maybe it was something from previous iterations